client/src/modules/chapters/pages/chapterPage.tsx

```diff
@@ -154,15 +155,21 @@ export const ChapterPage: NextPage = () => {
   return (
     <VStack>
       <Stack w={['90%', '90%', '60%']} maxW="600px" spacing={6} mt={10} mb={5}>
-        <Image
-          boxSize="100%"
-          maxH="300px"
-          src={data.chapter.banner_url ? data.chapter.banner_url : ''}
-          alt=""
-          borderRadius="md"
-          objectFit="cover"
-          fallbackSrc="https://cdn.freecodecamp.org/chapter/puppy-small.jpg"
-        />
+        {data.chapter.banner_url === '' ||
+        data.chapter.banner_url === null ? null : (
```

Comment:
```suggestion
        {data.chapter.banner_url && (
```
The truthiness of `banner_url` is the only thing that matters here, so we can simplify a bit.